dotnet/src/SemanticKernel.UnitTests/KernelTests.cs

```diff
@@ -120,47 +120,49 @@ public async Task RunAsyncHandlesPreInvocationAsync(int pipelineCount)
     {
         // Arrange
         var sut = new KernelBuilder().Build();
-        var myPlugin = new Mock<MyPlugin>();
```

Comment:
While the changes in this file are nice anyway, the impetus for changing them was because there appears to be an issue on .NET Framework with getting attributes on return parameters when the target object is coming from Mock.